models/contrib/nish/curve.py

```diff
@@ -205,7 +205,7 @@ class CurvePoolDepeggingAmount(DTO):
     amount_required: float
```

Comment:
according to the formula, desired_ratio needs to be less or equal to 1. It's better to put a check in DTO
`= DTOField(..., le=1.0)`